tests/pfcwd/conftest.py

```diff
@@ -209,14 +210,29 @@ def setup_dut_test_params(
 
 # icmp_responder need to be paused during the test because the test case
 # configures static IP address on ptf host and sends ICMP reply to DUT.
-@pytest.fixture(scope="module")
-def pause_icmp_responder(ptfhost):
-    icmp_responder_status = ptfhost.shell("supervisorctl status icmp_responder", module_ignore_errors=True)["stdout"]
-    if "RUNNING" not in icmp_responder_status:
-        yield
-        return
-    ptfhost.shell("supervisorctl stop icmp_responder", module_ignore_errors=True)
+@pytest.fixture(scope="module", autouse=True)
+def pfcwd_pause_service(ptfhost):
+    needs_resume = {"icmp_responder": False, "garp_service": False}
+
+    out = ptfhost.shell("supervisorctl status icmp_responder", module_ignore_errors=True).get("stdout", "")
+    if 'RUNNING' in out:
+        needs_resume["icmp_responder"] = True
+        ptfhost.shell("supervisorctl stop icmp_responder")
+
+    out = ptfhost.shell("supervisorctl status garp_service", module_ignore_errors=True).get("stdout", "")
+    if 'RUNNING' in out:
+        needs_resume["garp_service"] = True
+        ptfhost.shell("supervisorctl stop garp_service")
+
+    logger.debug("pause_service needs_resume {}".format(needs_resume))
```

Comment:
we also need to flush the arp/ndp table

Reply:
The script will call the "remove_ip.sh" to flush ip addr and then do the ping cmd, is it necessary to explicitly flush the arp table, thanks.

Reply:
Update for flushing table on ptf and dut